src/main/java/mate/academy/springbootwebgreqit/model/Book.java

```diff
@@ -0,0 +1,77 @@
+package mate.academy.springbootwebgreqit.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+@Entity
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    private String isbn;
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```

Comment:
add column constraints  (nullability, length etc)

Reply:
still not fixed